pkg/controller/middleware/auth.go

```diff
@@ -29,11 +29,10 @@ import (
 	"github.com/gorilla/mux"
 )
 
-// RequireAuth requires a user to be logged in. It also ensures that currentUser
-// is set in the template map. It fetches a user from the session and stores the
-// full record in the request context.
+// RequireAuth requires a user to be logged in. It also fetches and stores
+// information about the user on the request context.
 func RequireAuth(cacher cache.Cacher, authProvider auth.Provider, db *database.Database, h *render.Renderer, sessionIdleTTL, expiryCheckTTL time.Duration) mux.MiddlewareFunc {
-	cacheTTL := 15 * time.Minute
+	cacheTTL := 30 * time.Minute
```

Comment:
just wondering any particular reason for the cacheTTL doubling?

Reply:
I set it to 0 during testing, then changed it back. I apparently changed it back to the wrong value, but we could realistically cache this for 10000 days and it wouldn't matter because the gorm callbacks clear the cache when the key is changed or revoked.